subworkflows/local/preprocess_rnaseq/main.nf

```diff
@@ -0,0 +1,252 @@
+import groovy.json.JsonSlurper
```

Comment:
This is the most important thing to review- a subworkflow comprising preprocessing from rnaseq.

Once reviewed and approved I'll PR it to modules so we can factor it out of both rnaseq and riboseq.

Reply:
I would love that sooo much